src/foundry/client/apps/templates.d.mts

```diff
@@ -295,6 +261,47 @@ declare global {
       choices: Record<string, string> | Array<string>,
       options: HandlebarsHelpers.SelectOptionsOptions,
     ): Handlebars.SafeString;
+
+    /**
+     * Convert a DataField instance into an HTML input fragment.
+     * @param field   - The DataField instance to convert to an input
+     * @param options - Helper options
+     */
+    static formInput(
+      field: foundry.data.fields.DataField.Any,
+      options: {
+        // TODO: Derive valid hash options from the provided field
```

Comment:
Did you mean to resolve this before opening the PR?

Reply:
It's going to require implementing DataField#toInput, so not this PR; I'll add a comment to the fields ticket.